utils.js

```diff
@@ -398,8 +398,8 @@ function isDomElement(thing) {
  * execute any of its inline scripts.
  */
 function staticDom(html) {
-    const {jsdom} = require('jsdom');
-    return jsdom(html, {features: {ProcessExternalResources: false}});
+    var parser = new DOMParser();
```

Comment:
I've read that setting the script types to "application/json" would prevent them from loading. Do you think this is reasonable, or might there be a better way?